src/utils/cancelRequests.ts

```diff
@@ -39,19 +41,24 @@ export const getAxiosReqParams = (
     cache: defaultCache,
   };
 
-  if (!reqConfig) {
+  const reqConfigWithDefault = {
+    ...getDefaultRequestsConfig(),
+    ...reqConfig,
+  };
+
+  if (!reqConfigWithDefault) {
```

Comment:
`reqConfigWithDefault` will never be null or undefined so it should also probably check if object has at least one key. Not that it matters.

Reply:
~~I agree it doesn't... but anyway, fixed the condition.~~ Since this condition never triggered before, and since it doesn't add anything, I have simply removed it now. Thank you for bringing it to my attention! Fixed.